Mon, 30 Dec 2019 09:30:40 - INFO - Connect to DB
Mon, 30 Dec 2019 09:30:41 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 09:30:41 - INFO - Failed on TRIZMA DB import at: 2019-12-30_09-25-15, for following calling party: 4915227399439, for agent id: trz_sstankovic, with error:(pyodbc.DataError) ('22001', '[22001] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]String or binary data would be truncated. (8152) (SQLExecDirectW)')
[SQL: insert into records values('CC2Servicenummer','Belgrad','Nicht übermittelt','trz_sstankovic',                                  '4915227399439','','Nicht übermittelt','4915227399439_Saturday, December 21, 2019_4:13:51 PM')]
(Background on this error at: http://sqlalche.me/e/9h9h)
Mon, 30 Dec 2019 09:30:41 - INFO - 123TV DB import
Mon, 30 Dec 2019 09:38:25 - INFO - Connect to DB
Mon, 30 Dec 2019 09:38:25 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 09:38:25 - INFO - 123TV DB import
Mon, 30 Dec 2019 09:40:09 - INFO - Connect to DB
Mon, 30 Dec 2019 09:40:09 - INFO - TRIZMA DB import
Mon, 30 Dec 2019 09:40:09 - INFO - 123TV DB import
